pkg/monitoring/metrics/virt-handler/domainstats/dirty_rate_metrics_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package domainstats
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/client-go/api"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+
+	"kubevirt.io/kubevirt/pkg/monitoring/metrics/testing"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/stats"
+)
+
+var _ = Describe("dirty rate metrics", func() {
+	Context("on Collect", func() {
+		var (
+			vmiStats  *VirtualMachineInstanceStats
+			vmiReport *VirtualMachineInstanceReport
+		)
+		const (
+			mb = 1024 * 1024
+		)
```

Comment:
nit define this in pkg/monitoring/metrics/virt-handler/domainstats/dirty_rate_metrics.go
and use it also in
`dirtyRateInBytesPerSecond := dirtyRate.MegabytesPerSecond * 1024 * 1024`

didn't comment there because the variable names made it explicit, but if we are using it here, we can use a variable for both

Reply:
TBH I think that the variable re-use in this scenario does not worth the price of introducing a new dependency. In addition, `MB == 1024^2` is well known and will never change :) so I would vote for it to be inlined.

Reply:
I've removed the `mb` const from here as well.